spec/controllers/stocks_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe StocksController do
```

Comment:
there should be a _spec_ for the search function, and test both the results and no-results found scenarios.